src/components/Breadcrumbs.js

```diff
@@ -0,0 +1,63 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { css } from '@emotion/react';
+import { Link } from '@newrelic/gatsby-theme-newrelic';
+
+const Breadcrumbs = ({ segments, separator }) => {
+  return (
+    <div
+      css={css`
+        margin: 2em 0;
+        > :first-child {
+          margin-left: 0;
+        }
+        > * {
+          margin-left: 0.5em;
+        }
+      `}
+      aria-label="breadcrumb"
+    >
+      {segments.map((segment) => {
+        const elem = segment.url ? (
+          <Link to={segment.url}>{segment.name}</Link>
```

Comment:
One thing I thought of after reviewing this yesterday: should we _always_ present the last breadcrumb segment as a `<span>` rather than a link? There's not much sense in linking to the page the user is on and, if the component handles that, we wont need to do weird stuff like `url: null` (like we're doing in `QuickstartDetails.js`)

Reply:
yeah i thought about that. i know sometimes tech docs folks liked having things like breadcrumbs be links so they could copy/paste the link. and when i looked at google's material design for inspiration they used a link for the last segment sometimes, so thought maybe there's something to it and left it as optionally available.